tests/installation/bootloader_zkvm.pm

```diff
@@ -18,17 +18,20 @@ use bootloader_setup;
 use registration;
 use testapi;
 use utils qw(OPENQA_FTP_URL type_line_svirt save_svirt_pty);
+use ntlm_auth;
 
 sub set_svirt_domain_elements {
     my ($svirt) = shift;
 
     if (!get_var('BOOT_HDD_IMAGE') or (get_var('PATCHED_SYSTEM') and !get_var('ZDUP'))) {
         my $repo = "$utils::OPENQA_FTP_URL/" . get_required_var('REPO_0');
-        my $cmdline = get_var('VIRSH_CMDLINE') . " install=$repo ";
+        $repo = get_var('MIRROR_HTTP') if get_var('NTLM_AUTH_INSTALL');
+
         my $name = $svirt->name;
 
+        my $ntlm_proxy = $ntlm_auth::ntlm_proxy;
+        my $cmdline = get_var('VIRSH_CMDLINE') . " $ntlm_proxy install=$repo";
```

Comment:
If `$ntlm_proxy` is undefined, what happens here?

Reply:
Good question, it will go to the default installation process.

Reply:
Is there a warning message in log? such as 'using an undefined var'

Reply:
Use of uninitialized value $ntlm_proxy in concatenation (.) or string at /var/lib/openqa/pool/24/os-autoinst-distri-opensuse/lib/bootloader_pvm.pm line 104.